arch/arm/core/Kconfig

```diff
@@ -26,6 +26,19 @@ config CPU_CORTEX_M
 	help
 	  This option signifies the use of a CPU of the Cortex-M family.
 
+config CPU_CORTEX_R
+	bool
+	# Omit prompt to signify "hidden" option
+	default n
```

Comment:
`n` is the implicit default value for `bool` symbols, so could remove the default.

That generates clearer documentation at http://docs.zephyrproject.org/reference/kconfig/index.html as well ("Implicitly defaults to `n`" instead of "default n if \<possibly complicated dependencies>").